pkgs/by-name/pc/pcsx2-bin/package.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  fetchurl,
+  makeWrapper,
+  stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "pcsx2";
+  version = "1.7.5779";
+
+  src = fetchurl {
+    url = "https://github.com/PCSX2/pcsx2/releases/download/v${finalAttrs.version}/pcsx2-v${finalAttrs.version}-macos-Qt.tar.xz";
+    hash = "sha256-QdYV63lrAwYSDhUOy4nB8qL5LfZkrg/EYHtY2smtZuk=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontPatch = true;
+  dontConfigure = true;
+  dontBuild = true;
+
+  sourceRoot = ".";
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/{bin,Applications}
+    cp -r "PCSX2-v${finalAttrs.version}.app" $out/Applications/PCSX2.app
+    makeWrapper $out/Applications/PCSX2.app/Contents/MacOS/PCSX2 $out/bin/pcsx2-qt
+    runHook postInstall
+  '';
+
+  meta = {
+    homepage = "https://pcsx2.net";
+    description = "Playstation 2 emulator; precompiled binary for MacOS x64, repacked from official website";
+    longDescription = ''
+      PCSX2 is an open-source PlayStation 2 (AKA PS2) emulator. Its purpose is
+      to emulate the PS2 hardware, using a combination of MIPS CPU Interpreters,
+      Recompilers and a Virtual Machine which manages hardware states and PS2
+      system memory. This allows you to play PS2 games on your PC, with many
+      additional features and benefits.
+    '';
+    license = with lib.licenses; [
+      gpl3Plus
+      lgpl3Plus
+    ];
+    mainProgram = "pcsx2-qt";
+    maintainers = with lib.maintainers; [
+      hrdinka
```

Comment:
@AndersonTorres I think I am the only Darwin maintainer, as @hrdinka  and @GovanifY were on the original Linux derivation when I added Darwin support.

https://github.com/NixOS/nixpkgs/pull/320122/files#diff-fcfb4cc512aebed71f6d491a7b51d67b72d2c7ee4fdfe90dbd97ed6d701742a9L148

Reply:
OK, I will remember this

Reply:
Done.